sds_data_manager/utils/stackbuilder.py

```diff
@@ -192,6 +193,40 @@ def build_sds(
         env=env,
     )
 
+    # I-ALiRT IOIS ECR
+    ialirt_ecr = ecr_stack.EcrStack(
+        scope,
+        "IalirtEcr",
+        env=env,
+        instrument_name="IalirtEcr",
+    )
+
+    # I-ALiRT IOIS DynamoDB
+    # ingest-ugps: ingestion ugps - 64 bit
+    # sct-vtcw: spacecraft time ugps - 64 bit
+    # src-seq-ctr: increments with each packet (included in filename?)
+    # ccsds-filename: filename of the packet
+    dynamodb_stack.DynamoDB(
```

Comment:
I think you'll need to rebase because Tenzin got rid of this stack. I'd suggest just directly making your DynamoDB Table in the I-ALiRT stack and not bothering with a separate Stack() construct.

We should put `on_demand=True` for now since it really won't be doing much of anything at first and doesn't need read/write capacity always on.